webapp/crashstats/api/tests/test_views.py

```diff
@@ -196,7 +192,10 @@ def test_hit_or_not_hit_ratelimit(self):
             # eventually you'll run into the limit there too.
             assert response.status_code == 429
 
-    def test_ProcessedCrash(self):
+
+class TestProcessedCrashAPI(BaseTestViews):
+    @mock.patch("crashstats.crashstats.models.ProcessedCrash.get_implementation")
```

Comment:
I took out the code that mocks all the implementations out before the tests run. I removed a vast majority of the mocking, but left this one in. Because of that, we have to add the mocking as a decorator.